magclassic/templates/emails/shifts/created.txt

```diff
@@ -0,0 +1,26 @@
+{{ attendee.first_name }},
+
+Thanks for signing up to volunteer at MAGClassic!  You're currently assigned to the {{ attendee.assigned_depts_labels|join_and }} department{{ attendee.assigned_depts_labels|length|pluralize }}, but let us know if you'd also like to work in any other departments.
+
+You can complete the MagClassic volunteer checklist at:
+{{ c.URL_BASE }}/signups/login
+
+This checklist includes: 
+- Confirming that you are coming to MAGClassic.
+- Any food restrictions you might have for our Staff Suite
+- Your t-shirt size for our merchandise department{% if attendee.hotel_eligible %}
+- Hotel room request (nights and roommate request){% endif %}
+- Shift signup information.
+
+If you have trouble logging in, you can verify/update your personal information at {{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }}
+
+You can add / drop / change your shifts anytime until 11:59pm on Tuesday, September 8th.
+
+Please let us know if you have any questions.
+
+Thank You,
+Stops@magfest.org
```

Comment:
silly tiny nitpick, we could use {c.STOPS_SIGNATURE} or whatever that was named here

don't... fix this.  it's fine.